src/components/menus/directory-tree-contextual-menu.js

```diff
@@ -380,6 +405,17 @@ const DirectoryTreeContextualMenu = (props) => {
             icon: <FolderSpecialIcon fontSize="small" />,
         });
 
+        if (!showMenuFromEmptyZone()) {
```

Comment:
You already have this condition in line 320, just add this code to this block

Reply:
i add it here so it will be the last icon in the toolbar.